src/main/java/menelaus/model/board/Piece.java

```diff
@@ -63,13 +65,34 @@ public void addTile(Tile tile) {
 		tiles.add(tile);
 	}
 	
-	public Piece(Point position) {
+	public Piece(Point position, PieceDrawer pieceDrawer) {
```

Comment:
I figured if we wanted to go the route without having PieceView class each piece would have to be able to draw itself, this is the way it worked in the WordMap example. I suppose we could just have the draw be implicitly a piece, then this wouldn't really be necessary.